create-anaconda-payload

```diff
@@ -72,9 +109,19 @@ def build_payload(image: str, output: str) -> None:
         )
 
         # Change directory to /mnt/sysimage/ and create archive
-        machine.execute("cd /mnt/sysimage && tar --selinux --acls --xattrs -zcvf /root/payload.tar.gz *", timeout=100)
+        machine.execute(
+            "cd /mnt/sysimage && tar --selinux --acls --xattrs -zcvf /root/webui-payload.tar.gz *",
+            timeout=100
+        )
+
+        # Create the payload RPM Spec file
+        machine.execute(f"echo \"{WEBUI_PAYLOAD_SPEC}\" > /root/webui-payload.spec")
```

Comment:
machine.write_file()

Reply:
I think the `machine.write("/root/webui-payload.spec", WEBUI_PAYLOAD_SPEC)` is the right syntax.

Reply:
Oops, sorry, yes. Mixed it up with `MachineCase.write_file()` :monkey_face: